src/write-batch.js

```diff
@@ -106,6 +106,20 @@ class WriteResult {
   get writeTime() {
     return this._writeTime;
   }
+
+  /**
+   * Returns true if this `WriteResult` is equal to the provided value.
```

Comment:
FWIW, I can't imagine a case where this is useful so we could probably omit it... but I don't care strongly.